HISTORY.rst

```diff
@@ -6,10 +6,25 @@ History
 .. to_doc
 
 ---------------------
-0.9.1.dev0
----------------------
-
-    
+0.10.0.dev0
+---------------------
+
+* Refactor shed tool functionality to be more modular, readable, and concise
+  (thanks to `@rhpvorderman`_).
+  `Pull Request 104`_
+* Add ``pytest``, enable coverage testing (thanks to `@rhpvorderman`_).
```

Comment:
Should all the testing be in the release note? I mean, it matters to us but what does the end user notice? (well less regressions probably, but we know that because we are developers. Our end users are not necessarily developers).

Reply:
@rhpvorderman I don't see reason why not to include it. Release notes have various people reading them and one of the reasons for notes is also to give credit (in this case to you ;)